mobile/src/main/res/layout/activity_vendor_home.xml

```diff
@@ -1,19 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
-<merge xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools">
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:orientation="vertical">
 
-    <fragment
-        android:id="@+id/vendor_map_fragment"
-        android:name="com.truckmuncher.app.vendor.VendorMapFragment"
+    <android.support.v7.widget.Toolbar xmlns:android="http://schemas.android.com/apk/res/android"
+        android:id="@+id/toolbar_actionbar"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        tools:context="vendor.VendorHomeActivity" />
+        android:layout_height="wrap_content"
+        android:background="?attr/colorPrimary"
+        android:minHeight="?attr/actionBarSize">
 
-    <fragment
-        android:id="@+id/vendor_home_fragment"
-        android:name="com.truckmuncher.app.vendor.VendorHomeFragment"
+    </android.support.v7.widget.Toolbar>
+
+    <FrameLayout
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        tools:context="vendor.VendorHomeActivity" />
+        android:layout_height="match_parent">
+
+        <fragment
+            android:id="@+id/vendor_map_fragment"
+            android:name="com.truckmuncher.app.vendor.VendorMapFragment"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            tools:context="vendor.VendorHomeActivity" />
 
-</merge>
+        <fragment
+            android:id="@+id/vendor_home_fragment"
+            android:name="com.truckmuncher.app.vendor.VendorHomeFragment"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            tools:context="vendor.VendorHomeActivity" />
```

Comment:
Move the context attribute to parent and add `tools:layout="@layout/fragment_vendor_home"` to this fragment definition